bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/i18n/netatmo.properties

```diff
@@ -344,6 +344,13 @@ channel-type.config.netatmo.live-stream-url.quality.option.low = Low Quality
 channel-type.config.netatmo.live-stream-url.quality.option.poor = Poor Quality
 channel-type.config.netatmo.live-stream-url.quality.option.high = High Quality
 
+# channel types
+
+channel-type.netatmo.siren-status.label = Status
+channel-type.netatmo.siren-status.description = Status of the siren
+channel-type.netatmo.siren-status.state.option.no_sound = Silent
+channel-type.netatmo.siren-status.state.option.sound = Alarm
```

Comment:
This block should be removed ?
You already have label and description added below and the two options are now obsolete.